blog/wyniki-badania-plac-w-komunikacji-technicznej-2023/index.mdx

```diff
@@ -0,0 +1,216 @@
+6. Jaka jest Twoja forma zatrudnienia?
+7. Jak duża jest Twoja firma?
+8. Jakiego rodzaju dokumentację tworzysz?
+9. Ile wynosi Twoje miesięczne wynagrodzenie brutto?
+10. Jak bardzo jesteś zadowolony/a ze swojej pracy w skali od 1 do 5 ?
+11. Jakie jest Twoje główne wykształcenie?
+
+Dodatkowo, pojawiły się dwa nowe pytania:
+
+1. Czy dla Twojego wynagrodzenia zastosowanie mają autorskie koszty uzyskania
+  przychodu (tzw. tax break)?
+2. W jakim modelu pracujesz - zdalnie czy hybrydowo?
+
+## Porównanie z poprzednimi latami
+
+Jeśli ciekawi Was co zmieniło się od naszego pierwszego badania, sprawdźcie zestawienie poniżej.
+
+<YearlyComparisonChart />
+
+## Pierwsze wnioski
+
+- W tym zebraliśmy aż 133 odpowiedzi! Bardzo nas cieszy, że tak wiele osób jest
+  zainteresowanych naszą ankietą. Ciekawe jaki to procent wszystkich tech writerów w Polsce.
+  🤔
+- Zarobki nadal mają tendencję wzrostową. 💰
+
+## Profesje
+
+Podobnie jak w latach poprzednich, większość osób biorących udział w ankiecie
+identyfikuje się jako Technical Writer (specjalista ds. dokumentacji
+technicznej). Pojawił się też bardzo ciekawy tytuł "Content Wrangler", ale
+zakwalifikowaliśmy go jako "technical writer".
+
+W odpowiedziach pojawiały się też feminatywy z jednoczesnym
+spolszczeniem, takie jak "technical writerka". Jako znawcy języka, jesteśmy bardzo ukontentowani tym faktem.
+
+<SurveyTable {...require('./data').professionalTitlesNumbersTableProps} />
+
+## Przygotowanie do zawodu
+
+Większość ankietowanych nauczyło się zawodu w pracy (87%).
+
+<SurveyTable {...require('./data').trainingTableProps} />
+
+12 osób wybrało opcję "Inny kurs lub certyfikat", oto ich odpowiedzi:
+
+<SurveyBulletedList
+  entries={require('./data').trainingBarChartDataset.filter(
+    (item) => item.count === 1
+  )}
+/>
+
+### Kurs ITCQF
+
+Co ciekawe, na trzecim miejscu pojawił się kurs ITCQF - aż 28 osób się nim
+pochwaliło. To duży skok w porównaniu z poprzednim rokiem, kiedy ten kurs
+wspomniało tylko 9 osób.
+
+<SurveyTable {...require('./data').itcqfTableProps} />
+
+Bardzo nas cieszy, że coraz więcej osób decyduje się na uczestnictwo w tym kursie, ale 
+mamy też świadomość, że ten udział procentowy we wszystkich odpowiedziach nie jest odzwierciedleniem rzeczywistości.
+Osoby, które przeszły kurs zapewne mają w swoich kręgach na mediach społecznościowych osoby powiązane z ITCQF i usłyszały o naszym portalu
+podczas szkolenia. Dzięki temu dotarli do informacji o ankiecie i prawdopodobnie dlatego tak sporo osób z kursem ITCQF wzięło w niej udział. 
+Gdybyśmy zebrali więcej odpowiedzi, prawdopodobnie udział procentowy osób z kursem ITCQF byłby dużo niższy.
+
+### Doświadczenie
+
+Jak widać poniżej, doświadczenie ma dość znaczący i pozytywny 
+wpływ na to jakie otrzymujemy wynagrodzenie.
+
+<SurveyTable {...require('./data').experienceAndEarningsTableProps} />
+
+### Wykształcenie
+
+Poniższa tabelka odpowiada na pytanie czy rodzaj wykształcenia wpływa na zarobki.
+
```

Comment:
W tabeli "Jakie jest Twoje glowne wyksztalcenie" niektore kwoty nie maja rozdzielenia czesci tysiecznej. Zamiast "9 300" jest "9300". Moze to taka konwencja ze tylko rozdzielacie gdy czesc tysieczna jest dwucyfrowa np.: "21 500"? I tak jesli patrzymy od gory do dolu i mamy kwoty z liczba jedno i dwucyfrowa w pozycji tysiecznej to tak nie beda w jednej linii (chyba ze spacja z przodu ;-)
Problem raczej natury estetycznej. Moze rozdzielanie w kazdym przypadku ulatwiloby skanowanie tekstu wzrokiem.